src/Pages/Login/Login.component.js

```diff
@@ -1,10 +1,60 @@
 import React, { Component } from "react";
+import { Link } from "react-router-dom";
 
 import "./Login.styles.scss";
 
 class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      userId: "",
+      userPwd: "",
+    };
+  }
+
+  handleIdPwd = (e) => {
+    this.setState({ [e.target.name]: e.target.value });
+  };
+
   render() {
-    return <div></div>;
+    return (
+      <div className="Login">
+        <div className="login-container">
+          <h3 className="login-title">로그인</h3>
+          <form className="login-form">
+            <input
+              className="login-input id"
+              name="userId"
+              type="text"
+              placeholder="아이디를 입력해주세요"
+              onChange={this.handleIdPwd}
+            />
+            <input
+              className="login-input pwd"
+              name="userPwd"
+              type="text"
+              placeholder="비밀번호를 입력해주세요"
+              onChange={this.handleIdPwd}
+            />
+            <div className="check-and-search">
+              <div className="check">
+                <label htmlFor="connect-type">
+                  <input type="checkbox" name="connect" id="connect-type" />
```

Comment:
```suggestion
                <label>
                  <input type="checkbox" name="connect" />
```
label 태그로 input 태그를 감싸게 되면 굳이 htmlFor와 id가 필요 없게 됩니다

Reply:
네! 수정하겠습니다. 대충 알고 사용하는 일이 많아서 이런일이 생기나봐요 ㅠ